.github/workflows/main.yml

```diff
@@ -20,6 +20,15 @@ jobs:
           python -m pip install tox
       - name: Run tests
         run: tox
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v1
+        with:
+          flags: >-
+            GHA,
+            ${{ runner.os }},
+            ${{ matrix.python }}
+          functionalities: gcov
+          verbose: true # optional (default = false)
```

Comment:
My preference would be to start without any options unless the project specifically benefits from the options. Why enable verbose?